integrations/weaviate/tests/test_document_store.py

```diff
@@ -265,6 +265,7 @@ def test_to_dict(self, _mock_weaviate, monkeypatch):
                         "session_pool_connections": 20,
                         "session_pool_maxsize": 100,
                         "session_pool_max_retries": 3,
+                        "session_pool_timeout": 5,
```

Comment:
unrelated, needed to fix Weaviate tests and due to https://github.com/weaviate/weaviate-python-client/releases/tag/v4.9.0